languages.toml

```
@@ -17,7 +17,7 @@ bitbake-language-server = { command = "bitbake-language-server" }
 bufls = { command = "bufls", args = ["serve"] }
 cairo-language-server = { command = "cairo-language-server", args = [] }
 circom-lsp = { command = "circom-lsp" }
-cl-lsp = { command = "cl-lsp", args = [ "stdio" ] }
+cl-lsp = { command = "cl-lsp", args = [ "--stdio" ] }
```

```suggestion
cl-lsp = { command = "cl-lsp" }
```

we should be able to remove just this `args` part and it should start on stdio by default